ui/react-components/components/AppointmentEditor/AppointmentEditorService.js

```diff
@@ -1,7 +1,7 @@
-import {saveOrUpdateAppointment} from '../../api/appointmentsApi';
-import {saveRecurringAppointments} from "../../api/recurringAppointmentsApi";
+import {getConflicts, saveOrUpdateAppointment} from '../../api/appointmentsApi';
+import {getRecurringConflicts, saveRecurringAppointments} from "../../api/recurringAppointmentsApi";
 
-const getFormattedProviders = providers => providers.forEach(provider => {
+const updateProviders = providers => providers.forEach(provider => {
```

Comment:
Can we have Provider model?

Reply:
We will handle this as a separate tech debt card.